<launch>
  <node pkg="pointcloud_crop" type="pointcloud_crop_node" name="pointcloud_crop_node" output="screen">
    <param name="sub_topic" type="string" value="/point_cloud" />
    <param name="pub_topic" type="string" value="/point_cloud_crop" />

    <param name="x_min" value="-1" />
    <param name="x_max" value="1" />

    <param name="y_min" value="-1" />
    <param name="y_max" value="1" />

    <param name="z_min" value="-1"/>
    <param name="z_max" value="1"/>

    <param name="marker" type="bool" value="false" />
  </node>
</launch>
